working/augmentation-libraries/feature-specification.md

```diff
@@ -525,6 +464,11 @@ augment class C {
 }
 ```
 
+An extension type declaration induces an implicit unnamed constructor that is
+complete. *This is because it implicitly initializes the representation field.
+That implies that you can't augment an extension type declaration's constructor
+to give it a body.*
+
```

Comment:
An extension type declaration also introduces a complete getter for the representation variable.